sdk/eventgrid/azure-eventgrid/CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 # Release History
 
-## 4.15.1 (Unreleased)
+## 4.16.0b1 (Unreleased)
 
 ### Features Added
 
-- Added a new EventGridClient that supports `publish_cloud_events`, `receive_cloud_events`, `acknowledge_cloud_events` , `release_cloud_events`, and `reject_cloud_events` operations.
-- Added new models to support these new operations on EventGridClient.
+- Added new features to the EventGridClient that supports `publish_cloud_events`, `receive_cloud_events`, `acknowledge_cloud_events` , `release_cloud_events`, and `reject_cloud_events` operations. These features include a `renew_cloud_event_locks` operation, as well as a `release_with_delay` parameter on the `release_cloud_events` operation.
+- Added new models to support these new features on EventGridClient.
```

Comment:
does a note need to be made about lock_token -> reject_options/acknowledge_options or no?

Reply:
will add